src/encoded/audit/reference_epigenome.py

```diff
@@ -97,15 +97,16 @@ def audit_reference_epigenome_assay_types_requirments(value, system):
         project_detail = 'required according to standards of Minimal IHEC Reference Epigenome.'
```

Comment:
Rebase to get tests passing.

Reply:
rebased

Reply:
thx, one test failing.  could be related to update?

src/encoded/tests/test_audit_reference_epigenome.py::test_reference_epigenome_with_required_assays FAILED